x/stake/new_stake_test.go

```diff
@@ -0,0 +1,230 @@
+// test BEPHHH: new stake mechanism
```

Comment:
The name `new_stake` seems to be not very clear. Maybe we can refine it.

Reply:
I'll change it to `bep159_test.go`